src/main/java/edu/kit/hci/soli/domain/User.java

```diff
@@ -12,6 +9,7 @@
  * The datamodel for a User as it is stored in the database
  */
 @Entity
+@Table(name = "soli_users")
```

Comment:
See other comment for reasoning

Reply:
If you are changing this, than please keep it consistent